notebooks/sprints/sprint19/530_rsclient_staging.ipynb

```diff
@@ -0,0 +1,520 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e3d48a1b",
+   "metadata": {},
+   "source": [
+    "## This demo showcases the implementation of user story 530\n",
+    "\n",
+    "All initiated jobs for the staging process should be stored in a persistent PostgreSQL database. This ensures that even if the rs-server-staging pod is restarted, users can still check the status of all previously submitted jobs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "513ee19d-3941-404e-8d48-3886f590280c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Auxip service: http://rs-server-adgs:8000\n",
+      "CADIP service: http://rs-server-cadip:8000\n",
+      "Catalog service: http://rs-server-catalog:8000\n",
+      "Staging service: http://rs-server-staging:8000\n"
+     ]
+    }
+   ],
+   "source": [
+    "import requests\n",
+    "import os\n",
+    "import pprint\n",
+    "import time\n",
+    "# Init environment before running a demo notebook.\n",
+    "from resources.utils import *\n",
+    "\n",
+    "pp = pprint.PrettyPrinter(indent=2, width=80, sort_dicts=False, compact=True)\n",
+    "session = requests.Session()\n",
+    "user = os.environ[\"JUPYTERHUB_USER\"] if cluster_mode else os.environ[\"RSPY_HOST_USER\"]\n",
+    "auxip_client, cadip_client, stac_client, staging_client = init_demo(owner_id = user)\n",
+    "if os.getenv(\"RSPY_LOCAL_MODE\") == \"1\":\n",
+    "    href = \"http://rs-server-cadip:8000\"\n",
+    "    href_adgs = \"http://rs-server-adgs:8000\"\n",
+    "    href_staging = \"http://rs-server-staging:8000\"\n",
+    "else:\n",
+    "    href = href_adgs = os.environ[\"RSPY_WEBSITE\"]\n",
+    "    href_staging = \"https://rsserverstaging.dev-rspy.esa-copernicus.eu\"\n",
+    "    session.cookies.set (\"session\", os.environ[\"RSPY_OAUTH2_COOKIE\"])\n",
+    "\n",
```

Comment:
Instead of all this you should just do: 
```
from resources.utils import *
init_demo(owner_id = user)
from resources.utils import * # reload the global vars again
```
It already handles all this automatically.